### ISSUE TITLE
Installation reverted to dark mode upon upgrade

### ISSUE THREAD

**Issue by markmhx** [blockstack/stacks-wallet#871]
My system and the previous wallet were / are set to light mode, but upon upgrade to the latest version (v4.4.0) the wallet reverted automatically to dark mode:

![image](https://user-images.githubusercontent.com/28991/126161032-3c4e1d74-30b9-4e0d-82ac-f1dfa39556a5.png)

![image](https://user-images.githubusercontent.com/28991/126161050-a6689b2e-39e0-43bc-b169-9636b6db686f.png)

When I click "use system", it corrects to light mode:

![image](https://user-images.githubusercontent.com/28991/126161077-79015530-4d4e-48cf-a6b4-9db8354ac1fa.png)

**Comment by beguene** [comment]
It’s working fine for me on MacOS
Here are the steps I followed to reproduce:
1. Set the OS to ‘Light’ mode
2. Download & Install the previous version https://github.com/blockstack/stacks-wallet/releases/tag/v4.3.1
3. Launch Stack Wallet: it’s in white mode as expected
4. Download & Install the latest version https://github.com/blockstack/stacks-wallet/releases/tag/v4.4.0 and replace the previous one
5. Launch Stack Wallet : it’s still in white mode as expected

**Comment by markmhx** [comment]
I never did step 1 explicitly in that I just rely on my OS default for light mode. Could that present a difference in handling?

**Comment by beguene** [comment]
I don't think so. I tried again and still no issue. I will close this then. Please reopen if you face it again.

**Comment by markmhx** [comment]
I ran into this again here: https://github.com/blockstack/stacks-wallet/pull/926#issuecomment-933314043
